src/routes/sdkv2/signer.ts

```diff
@@ -139,6 +148,112 @@ export class SDKv2Signer<N extends Network, C extends Chain>
     return txHashes;
   }
 
+  async signTypedData(
```

Comment:
Remove this function and use signTypedData() available from the wallet via the signer